### PR TITLE
Update Settings.cs // Better standard values for itemRecycleFilter

### PR DESCRIPTION
**Pull request by Spegeli**
Better standard values.
This items are filling up your bags, so remove them.

### CODE REVIEW

**Comment by FeroxRev** [comment]
Could you keep the PokeBalls? Perhaps at let say 150?

**Comment by Spegeli** [comment]
Added now: 

new KeyValuePair<ItemId, int>(ItemId.ItemPokeBall, 150),

File: PokemonGo.RocketAPI.Console/Settings.cs [hunk 1]
```diff
@@ -21,11 +21,16 @@ public class Settings : ISettings
         {
             get
             {
-                //Type and amount to keep
+                //Items to Recylce but keep X amount
```
Comment (navidemad) [inline PokemonGo.RocketAPI.Console/Settings.cs]: Recycle instead of Recylce

File: PokemonGo.RocketAPI.Console/Settings.cs [hunk 2]
```diff
@@ -21,11 +21,16 @@ public class Settings : ISettings
...
                 {
-                    new KeyValuePair<ItemId, int>(ItemId.ItemPokeBall, 50),
-                    new KeyValuePair<ItemId, int>(ItemId.ItemGreatBall, 50)
+                    new KeyValuePair<ItemId, int>(ItemId.ItemPokeBall, 150),
+                    new KeyValuePair<ItemId, int>(ItemId.ItemPotion, 5),
+                    new KeyValuePair<ItemId, int>(ItemId.ItemSuperPotion, 5),
+                    new KeyValuePair<ItemId, int>(ItemId.ItemHyperPotion, 5),
+                    new KeyValuePair<ItemId, int>(ItemId.ItemMaxPotion, 5),
+                    new KeyValuePair<ItemId, int>(ItemId.ItemRevive, 5),
+                    new KeyValuePair<ItemId, int>(ItemId.ItemRazzBerry, 5)
```
Comment (navidemad) [inline PokemonGo.RocketAPI.Console/Settings.cs]: As right now we are using RazzBerry to catch great pokemon, the value of 5 is not too low ?

**Pull request closed without merging**